packages/perspective-viewer-d3fc/src/js/series/arcSeries.js

```diff
@@ -0,0 +1,21 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import {arc} from "d3";
+
```

Comment:
This "series" file doesn't seem quite aligned with the other series files. See other comments. I think there's a lot of code spread out around various folders that is more properly associated with drawing the "series" of the sunburst charts. It would be better to keep that code together rather than spreading it out.

I would be happy to see a sub-folder e.g. `/series/sunburst/etc.js` if there's too much series-related stuff to put in one file. Obviously it's rather different to the d3fc charts because the underlying series isn't implemented for us.